crates/ruff_linter/src/rules/flake8_use_pathlib/helpers.rs

```diff
@@ -138,6 +139,7 @@ pub(crate) fn is_file_descriptor(expr: &Expr, semantic: &SemanticModel) -> bool
     typing::is_int(binding, semantic)
 }
 
+#[allow(clippy::too_many_arguments)]
```

Comment:
Let's use `expect` instead of `allow`. That will remind us to remove it if we ever remove arguments.


```suggestion
#[expect(clippy::too_many_arguments)]
```